lvsfunc/comparison.py

```diff
@@ -594,21 +579,21 @@ def stack_vertical(*clips: vs.VideoNode) -> vs.VideoNode:
 
     :param clips: Clips for comparison (order is kept top to bottom)
 
-    :return: Returns a vertical stack of the clips
+    :return: Returns a vertical stack of the `clips`
     """
     return Stack(clips, direction=Direction.VERTICAL).clip
 
 
 def tile(**clips: vs.VideoNode) -> vs.VideoNode:
```

Comment:
I just now noticed that these convenience functions also/still use `*` and `**` instead of `clips: Union[Dict[str, vs.VideoNode], Sequence[vs.VideoNode]]`, which is a waste imo. Unless backwards-compatibility is important (not sure if even applicable), you should consider changing these to what `Comparer` uses in the constructor, since you're just forwarding that anyway. (You could use a type alias for this since you'll be using it quite a few times, I suppose.

Compare these:
```py
split = lvf.comparison.split(v1=src_a, v2=src_b, v1_again=src_a, v2_again=src_b, v1_again_again=src_a)
# vs
split = lvf.comparison.split([src_a, src_b, src_a, src_b, src_a])
```

If it would break compatibility and that is deemed important … maybe deprecate them?

Reply:
I did this intentionally, because most (or at least the ones I know of) comparison functions or functions dealing with multiple clips usually work like this: `func(clipa, clipb, *args)`. For the functions like stack, where clip boundaries will be apparent because, well it's stacking the left border against the right border of a (assuming you're using the *comparison* functions to compare stuff here) similar frame, I left them as just `*args` since doing `stack_vertical(clipa, clipb, clipc)` is probably easier for a basic script writer. For cases where clip boundaries or the order might be unclear, I put them as `**kwargs` so something like `split(BD=bdmv, TV=src1)` could be done. If the user has them in a list `clips=[...]` already, `func(*clips)` will work fine in the `*args` functions. If the user wants to do something special with it, they can just initialize the classes directly with the additional params like `label_alignment`.

Reply:
Not a fan of the inconsistency, personally.

Reply:
louis suggested a way to accept both types and error if both are given, which is something I'll look into.

Regarding kwargs here, it's fine because it puts them into a dict before sending them to the class constructor, so no keyword collision here.

Reply:
Made it so all the functions accept both positional arg clips and named kwarg clips, and errors out if both types of args are given.